src/app/_translate/lang-en.ts

```diff
@@ -62,6 +62,7 @@ export const LANG_EN_TRANS = {
   ftppassword: 'FTP/HTTP password',
   ftpuser: 'FTP/HTTP user',
   fullhistory: 'See full processing history',
+  gaperror: '406 Plugin Execution Not Allowed',
```

Comment:
Is this message indeed correct? The story description mentions a message "This step is not allowed in this position. " as well.